index.html

```diff
@@ -2,108 +2,104 @@
 <html class="no-js" lang="en-us">
 
 <head>
-  <meta charset="utf-8"/>
-  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>NodeDay</title>
-
-  <link rel="stylesheet" href="2015-digitas/css/bootstrap.min.css">
-  <link rel="stylesheet" href="2015-digitas/css/app.css" type="text/css"/>
-
-  <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
-  <!--[if lt IE 9]>
-  <script src="2015-digitas/js/html5shiv.min.js"></script>
-  <script src="2015-digitas/js/respond.min.js"></script>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>NodeDay</title>
+
+    <link rel="stylesheet" href="2015-digitas/css/bootstrap.min.css">
+    <link rel="stylesheet" href="2015-digitas/css/app.css" type="text/css" />
+
+    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
+  <script src="js/html5shiv.min.js"></script>
+  <script src="js/respond.min.js"></script>
```

Comment:
Good catch @rfeathers. We'll take care of it in the morning. Mike will let you know when the PR is updated. Cheers.